src/Core/src/Core/IGraphicsView.cs

```diff
@@ -16,5 +17,37 @@ public interface IGraphicsView : IView
 		/// Informs the canvas that it needs to redraw itself.
 		/// </summary>
 		void Invalidate();
+
+		/// <summary>
+		/// Occurs when the the surface received a touch event.
+		/// </summary>
+		/// <param name="e">Provides data for the Touch event.</param>
+		void OnTouch(TouchEventArgs e);
+	}
+
+	public enum TouchAction
```

Comment:
We could make these different methods instead to avoid more touch actions/events that might collide with core in the future...

- OnTouchStarted()
- OnTouchMoved()
- OnTouchReleased()
- OnTouchCancelled()

?

Reply:
Like this: https://github.com/dotnet/Comet/blob/dev/src/Comet.Skia/SkiaView.cs#L72-L94

Reply:
https://github.com/dotnet/maui/pull/4342 FYI started the idea here to discuss